### ISSUE TITLE
1.44 not seeing menu options for Turn on Preferences Sync

### ISSUE THREAD

**Issue by niemyjski** [microsoft/vscode#94801]
- VSCode Version: 1.44
- OS Version: macOS 12.15.4

Steps to Reproduce:

1. Update to latest stable vs code.
2. Click gear icon in lower left and there is no option for `Turn on Preferences Sync`.

I also don't see any settings when searching settings. The only ones I see are these: 
![image](https://user-images.githubusercontent.com/1020579/78906766-65f35780-7a45-11ea-8bd6-bc52244f2eaf.png)

Does this issue occur when all extensions are disabled?: Yes/No
Yes

**Comment by guobing1993** [comment]
![image](https://user-images.githubusercontent.com/22333388/78908748-206e7300-7ab5-11ea-88f0-8d778fbd8a65.png)

**Comment by RMacfarlane** [comment]
As @guobing1993 points out, this is only available in the Insiders version of VSCode. You can try it out here: https://code.visualstudio.com/insiders/

**Comment by niemyjski** [comment]
This is the release notes for a stable release channel.. This is just
confusing, insiders is a completely different download/install. That's like
me offering two products but putting product b release notes in product a's
release notes. It's highlighted and clickable along with all those major
features that are available. I've known this feature was in insiders for a
long while. That's why I got really happy when I saw it in the stable
release notes.
